Kira_iris_regression.R

```diff
@@ -0,0 +1,74 @@
+iris	# Fisherʻs iris dataset
+dim(iris)	# dimensions
+names(iris)
+
+## Renaming iris with more convenient names
+## Accessing vectors within iris, saving as separate objects
+
+names(iris) <- c("sepalL", "sepalW",  "petalL", "petalW", "species")
+species <- iris$species   
+x <- iris$petalL
+y <- iris$petalW
+
+
+## Illustrating regression by graphing 
+
+fit <- lm( y ~ x )  # fitting the linear model
+b <- coef( fit ) # grabbing the coefficients
+int <- b[1] # intercept
+slope <- b[2] # slope
+
+plot(x, y) # plotting the data points
+abline(fit) # drawing the regression line on the plot
+# look up help for ?abline and figure out how to use 
+# abline by specify slope and intercept.
```

Comment:
great descriptive notes!

Reply:
:)

Reply:
These are my words. They should have been removed and in general the script should have been cleaned up for the final draft. Marguerite